src/bridgestan.cpp

```diff
@@ -9,14 +9,14 @@ int bs_patch_version = BRIDGESTAN_PATCH;
 
 #include <sstream>
 
-bs_model_rng* bs_construct(const char* data_file, unsigned int seed,
+bs_model_rng* bs_construct(const char* data, unsigned int seed,
                            unsigned int chain_id, char** error_msg) {
   try {
-    return new bs_model_rng(data_file, seed, chain_id);
+    return new bs_model_rng(data, seed, chain_id);
   } catch (const std::exception& e) {
     if (error_msg) {
       std::stringstream error;
-      error << "construct(" << data_file << ", " << seed << ", " << chain_id
+      error << "construct(" << data << ", " << seed << ", " << chain_id
```

Comment:
I guess this is invalid if `data` is `NULL`?

Reply:
It seems that you can use `operator<<` with a null pointer but it terminates the string. E.g, you get an error which looks like:
`Error: construct(` \<EOL\>
which isn't great. 

It's easy to special-case, though.

Reply:
Sorry for all the pedantic comments recently.
Seems I get that way when I write too much rust. :-)

Reply:
Absolutely no need to apologize, I appreciate them. Every pedantic comment now is a future bug report -> pr -> patch release cycle we get to skip.

(coincidentally, this is why I like languages like Rust or OCaml with picky compilers. The downside is sometimes it ends up with me being far too trusting of my C++ compiler...)